processor/k8sattributesprocessor/internal/kube/kube.go

```diff
@@ -270,6 +270,19 @@ func (r *FieldExtractionRule) extractField(v string) string {
 	return ""
 }
 
+// RegexpCompile make pattern fully anchored
+func RegexpCompile(pattern string) (*regexp.Regexp, error) {
+	return regexp.Compile("^(?:" + pattern + ")$")
+}
+
+func RegexpMustCompile(pattern string) *regexp.Regexp {
```

Comment:
This should not be exported. And if it's used in tests only, let's move it to the test file

Reply:
Yes, i'll move to test file